database/migrations/2024_03_22_025035_change_group_admin_role_to_global-group-admin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Spatie\Permission\Models\Role as PermissionRole;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void {
+
+        PermissionRole::where('name', 'group admin')->update(['name' => 'global group admin']);
```

Comment:
Spatie has its own [permissions cache](https://spatie.be/docs/laravel-permission/v6/advanced-usage/cache), and I'm not sure our deploy script will clear it. Might be best practice for a case like this to also manually flush perms in this migration
```
app()->make(\Spatie\Permission\PermissionRegistrar::class)->forgetCachedPermissions();
```